CONTRIBUTING.md

```diff
@@ -430,6 +450,14 @@ See [Troubleshooting section](#troubleshooting) for steps you can make to clean
 Once you performed the first build, the images are rebuilt locally rather than pulled - unless you
 force pull the images. But you can force it using the scripts described below.
 
+## Automated image building
+
+You can also export one of the three variables to control the behaviour of image rebuilding:
+
+* Images will be automatically rebuild when needed: `ASSUME_YES="true"`
+* Old images are used even if rebuild is needed (useful when you have no connectivity): `ASSUME_NO="true"`
```

Comment:
Will re-use old images even if rebuild is necessary (useful for offline development)*